src/main/java/se/sundsvall/messaging/api/StatusAndHistoryResource.java

```diff
@@ -350,12 +350,16 @@ ResponseEntity<Statistics> getStats(
         value = {"/statistics/departments","/statistics/departments/{department}"},
         produces = { APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE }
     )
-    ResponseEntity<DepartmentStatistics> getDepartmentStats(
+    ResponseEntity<List<DepartmentStatistics>> getDepartmentStats(
```

Comment:
Consider wrapping the list in a wrapper class. 

Example,
If we need to add more data to the response later on. This current approach would break backwards compability and hence resulting in a new major.

If we instead use a wrapper class and add attributes to the wrapper, we only need to do a new minor version of the API.